Lib/inspect.py

```diff
@@ -173,6 +177,13 @@
 TPFLAGS_IS_ABSTRACT = 1 << 20
 
 
+@enum.global_enum
```

Comment:
I know it's specified by PEP-649, but I'd honestly prefer it if this were not a global enum. To me, `inspect.AnnotationsFormat.VALUE` (or `AnnotationsFormat.VALUE`) is much clearer than `inspect.VALUE`. There are loads of things that have values; `inspect.VALUE` could refer to anything. And `inspect.py` already has a huge module-level API.

Reply:
I don't disagree but for now I'm trying to implement the PEP as written. I think once the implementation is more complete we'll come up with more places where we may want to change the PEP's API.

Reply:
I've added a TODO as a sub-item in https://github.com/python/cpython/issues/119180#issue-2304664735 so we don't forget to revisit this later; marking this as resolved for now :-)